app/rdf/query-cube-termsets.ts

```diff
@@ -37,3 +38,49 @@ SELECT DISTINCT ?termsetIri ?termsetLabel WHERE {
     label: result.termsetLabel.value,
   }));
 };
+
+export const getCubeComponentTermsets = async (
+  iri: string,
+  options: {
+    locale: string;
+    sparqlClient: ParsingClient;
+  }
+): Promise<ComponentTermsets[]> => {
+  const { sparqlClient, locale } = options;
+  const query = `PREFIX cube: <https://cube.link/>
+PREFIX meta: <https://cube.link/meta/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX schema: <http://schema.org/>
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+
+SELECT DISTINCT ?dimensionIri ?dimensionLabel ?termsetIri ?termsetLabel WHERE {
+  VALUES (?iri) {(<${iri}>)}
```

Comment:
Yep. I think I just prefer this form because in query where ?iri is repeated, it is just easier when copy pasting the built query in lindas to change the ?iri at one place.